quantities.py

```diff
@@ -6,123 +6,119 @@
 
 
 quantity_definitions = {
-    "Quantity": {
```

Comment:
What's the motivation for removing `Quantity` as one of the root classes? This is something we want to preserve

Reply:
Sorry, I had no idea that `brick:Quantity` was supposed to be a root class. According to the [README.md](https://github.com/BrickSchema/brick-owl-dl/#classes), only five root classes were defined: `brick:Location`, `brick:Equipment`, `brick:Substance`, `brick:Tag`, `brick:Point`


I'm not very good at this but I'll take a stab at explaining why I still think this might be the right way. Please correct me if I'm wrong here. 

I used the following algorithm to find the root classes:
```
all_classes = all subjects of:
			?subject rdf:type owl:Class .

all_subclasses = all subjects of:
			?subject rdfs:subClassOf ?object .

# note: a root class can only have subclasses, it can't be a subclass of any other class
# the following will give us a set of classes that are not subclasses to any other class

root_classes = { all_classes } - { all_subclasses }
```
Without this change, this is what we'll get:
```
 root_classes = {
	brick:Location, 
	brick:Equipment,
	brick:Substance,
	brick:Tag,
	brick:Point
 }
```
With this change, this is what we'll get:
```
 root_classes = {
	brick:Location, 
	brick:Quantity,
	brick:Equipment,
	brick:Substance,
	brick:Tag,
	brick:Point
 }
```
So, in my opinion this change makes `brick:Quantity` a root class rather than destroying the fact that it is a root class.

Besides, some of the other root classes like [`brick:Substance`](https://github.com/BrickSchema/brick-owl-dl/blob/master/substances.py), [`brick:Location`](https://github.com/BrickSchema/brick-owl-dl/blob/master/location.py) and [`brick:Equipment`](https://github.com/BrickSchema/brick-owl-dl/blob/master/equipment.py) have done this in a similar manner.

Reply:
Yes, `brick:Quantity` should be a root class. The README is out of date